modules/backend/formwidgets/mediafinder/partials/_mediafinder.htm

```diff
@@ -1,6 +1,6 @@
 <?php if ($this->previewMode && !$value): ?>
 
-    <span class="form-control"><?= e(trans('backend::lang.form.preview_no_media_message')) ?></span>
+    <span class="form-control" <?= $field->getAttributes() ?>><?= e(trans('backend::lang.form.preview_no_media_message')) ?></span>
```

Comment:
Does any other previewMode partial file in October include `$field->getAttributes()`? My concern would be if they don't this would be unexpected behaviour for devs who might be using the attributes property to include data that they wouldn't include in preview mode if they knew that was happening.

Reply:
Good question. I've quickly checked all form widgets and some formfields and none of them do so. I'll remove it!